### PR TITLE
Upgrade to DW 13

### PR DESCRIPTION
**Pull request by fscala**

### CODE REVIEW

File: alchemy-client/pom.xml [hunk 1]
```diff
@@ -21,10 +21,18 @@
       <groupId>io.dropwizard</groupId>
       <artifactId>dropwizard-client</artifactId>
         <exclusions>
-            <exclusion>
-                <groupId>org.hibernate</groupId>
-                <artifactId>hibernate-validator</artifactId>
-            </exclusion>
+          <exclusion>
```
Comment (gtrog) [inline alchemy-client/pom.xml]: Why were these required?

Comment (fscala) [reply alchemy-client/pom.xml]: Version incompatibility 

hk2-locator:
        +-org.javassist:javassist:3.20.0-GA
        +-org.javassist:javassist:3.24.1-GA
org.apache.commons:
      +-org.apache.commons:commons-lang3:3.8.1
      +-org.apache.commons:commons-lang3:3.7
hibernate-validator: 
      +-org.hibernate:hibernate-validator:5.1.3.Final
      +-org.hibernate:hibernate-validator:5.4.3.Final
jersey-apache-connector:
      +-org.apache.httpcomponents:httpclient:4.5
       +-org.apache.httpcomponents:httpclient:4.5.10

Comment (gtrog) [reply alchemy-client/pom.xml]: What's pulling those in though?  It could mean we're pinning versions incorrectly elsewhere
